react/tic-tac-toe/src/app/screens/Game/components/MatchHistory/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import styles from './styles.module.scss';
+import matches from "~services/MatchesService";
+import { connect } from 'react-redux';
+import matchActions from "~redux/matches/actions";
+import Spinner from 'react-spinkit';
+import { arrayOf, func, boolean } from 'prop-types';
+import { matchPropType } from "~constants/propTypes.js"
+import {PLAYER_ONE, PLAYER_TWO, WINNER, MATCH_HISTORY_TITLE, GET_ERROR} from "~constants";
+
+class MatchHistory extends Component {
+
+    componentDidMount = () => {
+      const { toggleLoading, getMatches} = this.props;
+      toggleLoading();
+      matches.getMatches().then(response => getMatches(response.data));
+    }
+
+    renderLine = data => {
+      return (
+      <div key={data.id}>
```

Comment:
use `Fragment` here instead of divs https://reactjs.org/docs/fragments.html (they can be keyed)

Reply:
I think i need them in a div because i need them to be childs of a flex element for the styling